core-bundle/src/Command/DebugFragmentsCommand.php

```diff
@@ -86,6 +86,8 @@ static function ($carry, $item): int {
         foreach ($values as $k => $v) {
             if (\is_bool($v)) {
                 $v = $v ? 'true' : 'false';
+            } elseif ('nestedFragments' === $k && \array_key_exists('allowedTypes', $v)) {
+                $v = 'true (allowed types: '.implode(', ', $v['allowedTypes']).')';
```

Comment:
Maybe we could just implode if the value is an array? A debug output of `allowedFragments: alias, list` would work the same for me (and need less room)